stable/selenium/templates/firefox-deployment.yaml

```diff
@@ -78,8 +78,16 @@ spec:
             - name: TZ
               value: {{ .Values.firefox.timeZone | quote }}
             {{- end }}
+          volumeMounts:
+{{ if .Values.chrome.volumeMounts -}}
```

Comment:
Shouldn't this be ".Values.firefox.volumeMounts"?  Same issue on lines 83, 88, 89

Reply:
Thanks for noticing. I have changed it.